README.md

```diff
@@ -1,19 +1,6 @@
 ## pcre
 
 A simple regex library for [V](https://github.com/vlang/v).
-It wraps the venerable [PCRE](https://www.pcre.org/) library, so 
```

Comment:
The link to the original https://www.pcre.org/ should stay.

Reply:
There is a link in "Built With".

Reply:
Keep it both?

Reply:
I did not see it.

Reply:
No, I think one is enough.